source/cloud/azure/azure-mnmg.md

```diff
@@ -0,0 +1,70 @@
+# Multi-node, multi-GPU Cluster on Azure
+
+## Create a Cluster using Dask Cloud Provider
+
+The easiest way to setup a multi-node, multi-GPU cluster on Azure is to use [Dask Cloud Provider](https://cloudprovider.dask.org/en/latest/index.html).
+
+### 1. Install Dask Cloud Provider
+
+Dask Cloud Provider can be installed via `conda` or `pip`. The Azure-specific capabilities will need to be installed via the `[azure]` pip extra.
+
+```shell
+$ pip install dask-cloudprovider[azure]
+```
+
+### 2. Configure your Azure Resources
+
+Set up your Azure Resouce Group, Virtual Network, and Security Group according to [Dask Cloud Provider instructions](https://cloudprovider.dask.org/en/latest/azure.html#authentication).
```

Comment:
Nit: Could you link `Resource Group`, `Virtual Network` and `Security Group` to their respective Azure docs pages?